orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -79,6 +79,8 @@ def __init__(self, module, debug_options=None):
             # ORTModule.
             _utils.check_for_name_collisions_and_bind_methods_to_ortmodule(self, module)
 
+            self._modules = module._modules
+
```

Comment:
A solution to this problem is to `del self._modules`. If `ORTModule` does not have an attribute called `_modules`, all refrences to `ort_model._modules` will be forwarded to the underlying torch_model due to [ORTModule's __getattr__ implementation](https://github.com/microsoft/onnxruntime/blob/a5eb616819584d8f15ca21124e3892e9f209abc4/orttraining/orttraining/python/training/ortmodule/ortmodule.py#L275-L283).
If you go down this path, make sure to `self._modules` after `self._is_initialized = True`.

Reply:
Thanks a lot! Modified as your suggestion.